### PR TITLE
New D-TACQ devices for streaming and transient captures. (rebase)

### PR DESCRIPTION
**Pull request by zack-vii**
#1975

### CODE REVIEW

**Comment by zack-vii** [comment]
we should not ship https://github.com/petermilne/acq400_hapi
at least not as proper part but as submodule.
I would suggest to throw a warning if module is not found mentioning this url

**Comment by zack-vii** [comment]
still should be pyhton3 ready

**Comment by zack-vii** [comment]
retest this please

**Comment by zack-vii** [comment]
given that acq400_hapi is also used by acq400Devcies it is advised not to ship in at submodule of HTSDevcies.

also packaging need to be updated. we should combine all pure python pydevices into package mdsplus-pydevices.

**Comment by zack-vii** [comment]
fixed acq400Devices so they would work with and without acq400_base also fixed Queue

File: pydevices/acq400Devices/acq1001_st.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```
Comment (joshStillerman) [inline pydevices/acq400Devices/acq1001_st.py]: The previous is 'more correct'. The interpreter does not necessarily live a /usr/bin/python

File: pydevices/acq400Devices/acq1001_tr.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
```
Comment (joshStillerman) [inline pydevices/acq400Devices/acq1001_tr.py]: same comment as previous

File: pydevices/acq400Devices/acq2106_st.py [hunk 3]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
```
Comment (joshStillerman) [inline pydevices/acq400Devices/acq2106_st.py]: ditto

File: pydevices/acq400Devices/acq2106_tr.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
```
Comment (joshStillerman) [inline pydevices/acq400Devices/acq2106_tr.py]: again

**Review by joshStillerman — APPROVED** [review]

**Pull request merged**